### ISSUE TITLE
Get wrapped component

### ISSUE THREAD

**Issue by edimitchel** [eugene-manuilov/react-gettext#11]
While using withGettext method in order to internationalize the component, this librairy hide the concerning component from react application.
I don't find any way to get this component (eg. to use refs and method hosted inside).

Have you a suggestion or can you confirm that is currently not possible ?

**Comment by edimitchel** [comment]
May be something to see [here](hoist-non-react-statics) ?

**Comment by edimitchel** [comment]
[Another i18n library](https://react.i18next.com/components/translate-hoc.html#the-translate-hoc-can-take-a-few-options) provides an option called `withRefs` that can retrieve wrapper component by the `getWrappedInstance()` method.

**Comment by eugene-manuilov** [comment]
Hey @edimitchel,

thanks for asking for it. I have added support for it and released `0.4.0` version. Now you can pass 3rd parameter to the `withGettext` higher-order function to enable it:

```js
// popup.js
class Popup extends Component {

    save() {
        ... 
    }

}

export default withGettext(..., ..., { withRef: true })(Popup);
```

```js
// application.js

class Application extends Component {

    doSomething() {
        this.refs.popup.getWrappedComponent().save();
    }

    render() {
        return (
            <Popup ref="popup" />
        );
    }

}
```

Please, update to 0.4.0 and let me know if it helps to you.
